pilota-build/src/lib.rs

```diff
@@ -161,6 +165,11 @@ impl<MkB, P> Builder<MkB, P> {
         self
     }
 
+    pub fn with_split(mut self) -> Self {
```

Comment:
Maybe we can add an bool arg like `change_case` below? In which case we can both opt-in or opt-out this feature.